src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -1595,7 +1595,7 @@ bool main_thread::handle_1905_1_message(ieee1905_1::CmduMessageRx &cmdu_rx,
         return handle_1905_autoconfiguration_response(cmdu_rx, src_mac);
     }
     default: {
-        LOG(WARNING) << "Unknown 1905 message received. Ignoring";
+        // LOG(WARNING) << "Unknown 1905 message received. Ignoring";
```

Comment:
Remove completely instead of commenting out. Or replace with a comment saying that a warning should be printed as soon as all messages are supposed to be handled, or something like that.

Reply:
ok